server.js

```diff
@@ -1,9 +1,27 @@
-const express = require('express');
-const server = express();
+const http = require('http');
+const WebSocket = require('ws');
 const chalk = require('chalk');
 const constructXML = require('./src/junitFormatter');
 
-server.use(express.json());
+// Initialize a server
+const server = http.createServer();
+
+// Setup local variables for server
+server.locals = { appBooted: false};
+
+// Initialize a WebSocket Server instance
+const wss = new WebSocket.Server({server});
+
+// Setup the wanted behaviour for specific socket events
```

Comment:
Can you make this comment a little less generic? i.e. what's the behaviour and what are the events?